src/styles/variable.scss

```diff
@@ -0,0 +1,6 @@
+$main: #060B35;
+$secondary: #616070;
+$backimg: url(/src/images/star.svg);
+$activestar: url(/src/images/star-active.svg);
```

Comment:
use kebab-case

Reply:
also you use this value only in one place
you will not reuse it, so there is no point in removing this variable